### PR TITLE
Add __next__ to FakeFileWrapper

### LINKED ISSUE #485: FakeFileWrapper didn't implement `__next__`
**Describe the bug**

FakeFileWrapper didn't implement `__next__`

**How To Reproduce**

* next(f)

raise:

```
        with open(input_fn) as f:
>           student_ids = next(f).strip().split(',')
E           TypeError: 'FakeFileWrapper' object is not an iterator
```

**Your enviroment**
Please run the following and paste the output.
```bash
python -c "import platform; print(platform.platform())"
python -c "import sys; print('Python', sys.version)"
python -c "from pyfakefs.fake_filesystem import __version__; print('pyfakefs', __version__)"
```
```
Linux-4.20.0-rc1-x86_64-with-arch
Python 3.7.3 (default, May  3 2019, 11:10:59) 
[GCC 8.2.1 20181127]
pyfakefs 3.6
```

### PR DESCRIPTION
**Pull request by mlouielu**
Fix #485

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -16,24 +16,25 @@ The release versions are PyPi releases.
 #### Fixes
   * fake `DirEntry` now implements `os.PathLike` in Python >= 3.6
     (see [#483](../../issues/483))
-  * fixed incorrect argument name for `os.makedirs` 
+  * fixed incorrect argument name for `os.makedirs`
     (see [#481](../../issues/481))
   * avoid pytest warning under Python 2.7 (see [#466](../../issues/466))
+  * add __next__ to FakeFileWrapper
```
Comment (mrbean-bremen) [inline CHANGES.md]: Can you please add a reference to the issue like in the other entries?

File: pyfakefs/fake_filesystem.py [hunk 2]
```diff
@@ -4988,6 +4988,11 @@ def __iter__(self):
             self._raise('File is not open for reading')
         return self._io.__iter__()
 
+    def __next__(self):
+        if not self._read:
+            self._raise('File is not open for reading')
+        return self._io.__next__()
+
```
Comment (mrbean-bremen) [inline pyfakefs/fake_filesystem.py]: Can you please add a line to make this work in Python 2? Something like:
```python
if sys.version_info < (3, ):
    next = __next__
```

File: pyfakefs/tests/fake_open_test.py [hunk 3]
```diff
@@ -188,6 +188,19 @@ def test_iterate_over_file(self):
             result = [line.rstrip() for line in fake_file]
         self.assertEqual(contents, result)
 
+    def test_next_over_file(self):
+        contents = [
+            'Live long\n',
+            'and prosper\n'
+        ]
```
Comment (mrbean-bremen) [inline pyfakefs/tests/fake_open_test.py]: :)

File: CHANGES.md (hunk 1 above)
Comment (mlouielu) [reply CHANGES.md]: Addressed the point

File: pyfakefs/fake_filesystem.py (hunk 2 above)
Comment (mlouielu) [reply pyfakefs/fake_filesystem.py]: Addressed, also fix L4994 with `next(self._io)` for py2 compatiblity.

Comment (mrbean-bremen) [reply pyfakefs/fake_filesystem.py]: Thanks!

**Comment by mrbean-bremen** [comment]
Thanks for the contribution!

**Pull request merged**